<launch>
  <node pkg="pid_path_tracker" exec="pid_path_tracker_px4" output="screen">
    <param name="target_frame" value="base_link" />

    <param name="x_p"         value="0.5" />
    <param name="x_i"         value="0.0" />
    <param name="x_d"         value="0.0" />
    <param name="x_ff"        value="1.0" />
    <param name="x_d_alpha"   value="0.0" />
    <param name="x_min"       value="-0.2" />
    <param name="x_max"       value="0.2" />
    <param name="x_constant"  value="0.0" />
    
    <param name="y_p"         value="0.5" />
    <param name="y_i"         value="0.0" />
    <param name="y_d"         value="0.0" />
    <param name="y_ff"        value="1.0" />
    <param name="y_d_alpha"   value="0.0" />
    <param name="y_min"       value="-0.2" />
    <param name="y_max"       value="0.2" />
    <param name="y_constant"  value="0.0" />
    
    <param name="z_p"         value="2.0" />
    <param name="z_i"         value="0.0" />
    <param name="z_d"         value="0.0" />
    <param name="z_ff"        value="1.0" />
    <param name="z_d_alpha"   value="0.0" />
    <param name="z_min"       value="-0.25" />
    <param name="z_max"       value="0.25" />
    <param name="z_constant"  value="0.0" />
    
    <remap from="odometry" to="/ov/odom" />
    <remap from="trajectory" to="/cmd_trajectory" />
  </node>
</launch>